plugins/web/opentelemetry-instrumentation-document-load/src/instrumentation.ts

```diff
@@ -140,7 +154,7 @@ export class DocumentLoadInstrumentation extends InstrumentationBase<unknown> {
       addSpanNetworkEvent(rootSpan, PTN.LOAD_EVENT_END, entries);
 
       addSpanPerformancePaintEvents(rootSpan);
-
+      this._addCustomAttributes(rootSpan);
```

Comment:
I suggest calling this for all spans created by this instrumentation package. the callback has access to the span and can look into its properties if the custom attributes are required to be added  only to certain type of spans.

Reply:
Three different configs created to add attributes to the three different span types (document load, document fetch, and resource fetches)